features/guidance/staggered-intersections.feature

```diff
@@ -143,3 +143,21 @@ Feature: Staggered Intersections
             | a,m       | Oak St,Cedar Dr,Oak St,Oak St | depart,turn right,turn left,arrive |
             | m,a       | Oak St,Cedar Dr,Oak St,Oak St | depart,turn right,turn left,arrive |
 
+    Scenario: Staggered Intersection:
```

Comment:
@daniel-j-h does this test model what you had in mind? I can sort of imagine the collapsed silent scenario you were describing, but I can only get it to recognize the end `e` point here as part of the middle `jce` way, and not the branching `efg` way.